pkg/model/fileshare.go

```diff
@@ -95,6 +95,9 @@ type FileShareSpec struct {
 	// The uuid of the snapshot which the fileshare is created
 	SnapshotId string `json:"snapshotId,omitempty"`
 
+	// The name of the snapshot which the fileshare is created
+	SnapshotName string `json:"snapshotName,omitempty"`
+
```

Comment:
why add snapshot name in fileshare model?

Reply:
Because, currently we are creating logical volume for snapshot based on name. 
For ex: /dev/volume_group_name/snapshot_name.